src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,8 @@
+package duke;
+
+class DukeException extends RuntimeException {
+
+    public DukeException(String Message) {
+        super(Message);
+    }
```

Comment:
You could add a more descriptive message to be stored within the DukeException class